### ISSUE TITLE
The returned event does not have offsetX and offsetY property

### ISSUE THREAD

**Issue by huchun666** [Bernardo-Castilho/dragdroptouch#55]

**Comment by huchun666** [comment]
You can try it, there is indeed no offsetX and offsetY in the drag and drop events.
<img width="597" alt="WeChateac68cfc1ccba60ab5f818fa2e0f97fd" src="https://user-images.githubusercontent.com/28728583/221073547-4d80e228-587a-4688-8ac2-708994c3e3d9.png">

**Comment by huchun666** [comment]
I tried the new code below, still no offsetX and offsetY property.

**Comment by Bernardo-Castilho** [comment]
You are right. I looked more carefully at the [documentation](https://developer.mozilla.org/en-US/docs/Web/API/Touch) and realized **mouse** events have offsetX/Y properties but **touch** events do not. Touch events have screenX/Y, clientX/Y, and pageX/Y properties, but no offset.

I suppose we could calculate the offsets and stuff them into the event object, but I could not find any clear/simple way to do that. I found many [suggestions](https://stackoverflow.com/questions/17130940/retrieve-the-same-offsetx-on-touch-like-mouse-event), but none that seemed to be a clear winner.

My suggestion to you would be to do some research as well, and if you find a method that works well for your scenarios, call it in the event handler to obtain the values you want, e.g.:

```
function handleDragStart(e) {
    calculateEventOffsets(e);
    // now e should have offset values...
}
function calculateEventOffsets(e) {
    var bcr = e.target.getBoundingClientRect();
    e.offsetX = e.clientX - bcr.x;
    e.offsetY= e.clientY - bcr.y;
}
```
This sample uses one of the suggestions in the link above. It looks OK to me, but I have not tested it.

I hope it works for you.
